docs/references/vod/import.mdx

```diff
@@ -32,35 +31,18 @@ curl --location --request POST 'https://livepeer.studio/api/asset/import' \
 
 
 </TabItem>
-<TabItem value="js" label="NodeJS">
-
-```js
-app.post("https://livepeer.studio/api/asset/request-upload", {
-    headers: {
-      Authorization: "Bearer {api_key}",
-      "Content-Type": "application/json",
-    },
-    data: JSON.stringify({
-      name: "Example name",
-    }),
-  })
-  .then((res) => {
-    console.log(JSON.stringify(res.data));
-  })
-  .catch((error) => console.log(error));
-```
 
-</TabItem>
 <TabItem value="py" label="Fetch">
 
 ```js
-fetch("https://livepeer.studio/api/asset/request-upload", {
+fetch("https://livepeer.studio/api/asset/import", {
```

Comment:
The new URL is actually `/upload/url` now! `/request-upload` is still for upload from local storage.
```suggestion
fetch("https://livepeer.studio/api/asset/upload/url", {
```